lcftrans/src/translation.cpp

```diff
@@ -0,0 +1,1162 @@
+	t->addEntry(e);
+
+	e.original = Data::terms.normal_status;
+	e.context = "term";
+	e.info = "Term: normal_status";
+	t->addEntry(e);
+
+	e.original = Data::terms.exp_short;
+	e.context = "term";
+	e.info = "Term: exp_short";
+	t->addEntry(e);
+
+	e.original = Data::terms.lvl_short;
+	e.context = "term";
+	e.info = "Term: lvl_short";
+	t->addEntry(e);
+
+	e.original = Data::terms.hp_short;
+	e.context = "term";
+	e.info = "Term: hp_short";
+	t->addEntry(e);
+
+	e.original = Data::terms.sp_short;
+	e.context = "term";
+	e.info = "Term: sp_short";
+	t->addEntry(e);
+
+	e.original = Data::terms.sp_cost;
+	e.context = "term";
+	e.info = "Term: sp_cost";
+	t->addEntry(e);
+
+	e.original = Data::terms.attack;
+	e.context = "term";
+	e.info = "Term: attack";
+	t->addEntry(e);
+
+	e.original = Data::terms.defense;
+	e.context = "term";
+	e.info = "Term: defense";
+	t->addEntry(e);
+
+	e.original = Data::terms.spirit;
+	e.context = "term";
+	e.info = "Term: spirit";
+	t->addEntry(e);
+
+	e.original = Data::terms.agility;
+	e.context = "term";
+	e.info = "Term: agility";
+	t->addEntry(e);
+
+	e.original = Data::terms.weapon;
+	e.context = "term";
+	e.info = "Term: weapon";
+	t->addEntry(e);
+
+	e.original = Data::terms.shield;
+	e.context = "term";
+	e.info = "Term: shield";
+	t->addEntry(e);
+
+	e.original = Data::terms.armor;
+	e.context = "term";
+	e.info = "Term: armor";
+	t->addEntry(e);
+
+	e.original = Data::terms.helmet;
+	e.context = "term";
+	e.info = "Term: helmet";
+	t->addEntry(e);
+
+	e.original = Data::terms.accessory;
+	e.context = "term";
+	e.info = "Term: accessory";
+	t->addEntry(e);
+
+	e.original = Data::terms.save_game_message;
+	e.context = "term";
+	e.info = "Term: save_game_message";
+	t->addEntry(e);
+
+	e.original = Data::terms.load_game_message;
+	e.context = "term";
+	e.info = "Term: load_game_message";
+	t->addEntry(e);
+
+	e.original = Data::terms.file;
+	e.context = "term";
+	e.info = "Term: file";
+	t->addEntry(e);
+
+	e.original = Data::terms.exit_game_message;
+	e.context = "term";
+	e.info = "Term: exit_game_message";
+	t->addEntry(e);
+
+	e.original = Data::terms.yes;
+	e.context = "term";
+	e.info = "Term: yes";
+	t->addEntry(e);
+
+	e.original = Data::terms.no;
+	e.context = "term";
+	e.info = "Term: no";
+	t->addEntry(e);
+
+	// Read common events
+	bool has_message = false;
+	std::vector<std::string> choices;
+
+	for (size_t i = 0; i < Data::commonevents.size(); ++i) {
+		int evt_count = i + 1;
+
+		const std::vector<RPG::EventCommand>& events = Data::commonevents[i].event_commands;
+		for (size_t j = 0; j < events.size(); ++j) {
+			const RPG::EventCommand& evt = events[j];
+			int line_count = j + 1;
+
+			e.info = "Common Event " + std::to_string(evt_count) + ", Line " + std::to_string(line_count);
+			e.context = "event";
+
+			switch (evt.code) {
+			case Cmd::ShowMessage:
+				if (has_message) {
+					// New message
+					if (!e.original.empty()) {
+						e.original.pop_back();
+						t->addEntry(e);
```

Comment:
This adds the last message to the entries vector, but line 888 mixes its info with the info of the next line or event.